vendor/brightray/common/content_client.cc

```diff
@@ -8,9 +8,57 @@
 
 #include "base/strings/stringprintf.h"
 #include "base/strings/string_util.h"
+#include "content/public/common/cdm_info.h"
```

Comment:
please move this to atom_content_client. Brightray is deprecated and we want to get rid of these classes

Reply:
will do